config/Dockerfiles/tests.d/dummy/07-dummy-journal

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -xe
+
+# Verify --no-hooks and --ignore-failed-hooks on dummy provider
+# distros.exclude: none
+# providers.include: dummy
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+PINFILE="PinFile"
+WORKSPACE_PATH="docs/source/examples/workspaces/dummy-shell-hooks"
+TARGET="dummy_target"
+
+function clean_up {
+    set +e
+    linchpin -w . -p "${PINFILE}" -v destroy "${TARGET}"
+    D_RC=0
```

Comment:
Is this just trying to get the return code for linchpin destroy and exit if nonzero? This won't work if it is. You get $? after you do D_RC=0, so $? will always be 0. Why not just put ```if [ $? -ne 0 ]; then``` on line 18?